patterns/post-meta.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Title: Post Meta
+ * Slug: twentytwentythree/post-meta
+ * Categories: post meta
```

Comment:
This block pattern category does not exist. It could be registered, or another category can be picked and this can be used as keywords instead.

Reply:
Ah yeah, thanks. I meant to come back to this! I've changed the category to 'query', as this would always be used alongside a post query. If this doesn't work, maybe 'text' would work? And I've added 'post meta' as keywords 👍

Reply:
I think query is OK.. it is not a query loop but it can be placed in one to create new blog layouts.